vault/yaml/defaults.yaml

```diff
@@ -1,23 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
I wouldn't recommend using `<formula>/yaml/...` just yet; there have been arguments for and against doing this so it's probably better to avoid straying from the standard convention at the current time.